packages/app-form-builder/src/admin/components/FormEditor/Tabs/EditTab/FormStep/FormStep.tsx

```diff
@@ -54,24 +56,38 @@ export const FormStep = ({
     onDelete: () => void;
     onEdit: () => void;
     moveRow: (
```

Comment:
@adrians5j 
Would it be better if we move [these types](https://github.com/webiny/webiny-js/blob/19d9ab7794c32dbc181c8f0ad7f56a6d75291f60/packages/app-form-builder/src/admin/components/FormEditor/Tabs/EditTab/FormStep/FormStep.tsx#L52) into separate interface instead of having them in `FormStep` because it makes component bigger and in some way it makes it less readable.